src/ILICheck.Web/Controllers/StatusController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+using System.IO;
+
+namespace ILICheck.Web.Controllers
+{
+    [ApiController]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    public class StatusController : Controller
+    {
+        private readonly ILogger<StatusController> logger;
+        private readonly IValidatorService validatorService;
+        private readonly IFileProvider fileProvider;
+
+        public StatusController(ILogger<StatusController> logger, IValidatorService validatorService, IFileProvider fileProvider)
+        {
+            this.logger = logger;
+            this.validatorService = validatorService;
+            this.fileProvider = fileProvider;
+        }
+
+        /// <summary>
+        /// Gets the status information for the specified <paramref name="jobId"/>.
+        /// </summary>
+        /// <param name="version">The application programming interface (API) version.</param>
+        /// <param name="jobId">The job identifier.</param>
+        /// <returns>The status information for the specified <paramref name="jobId"/>.</returns>
+        /// <response code="200">The job with the specified <paramref name="jobId"/> was found.</response>
+        /// <response code="400">The server cannot process the request due to invalid or malformed request.</response>
+        /// <response code="404">The job with the specified <paramref name="jobId"/> cannot be found.</response>
+        [HttpGet("{jobId}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public IActionResult GetStatus(ApiVersion version, Guid jobId)
+        {
+            logger.LogTrace("Status for job id <JobId> requested.", jobId);
```

Comment:
REQ: Den Placeholder für die `jobId` korrekt einfügen
```suggestion
            logger.LogTrace("Status for job <{JobId}> requested.", jobId);
```